src/lib/parse/parseContextAttributeCompoundValue.h

```diff
@@ -0,0 +1,49 @@
+#ifndef SRC_LIB_PARSE_PARSECONTEXTATTRIBUTECOMPOUNDVALUE_H_
```

Comment:
The lib is old. Bdfore v2 it contained a few things that were common between jsonParse and xmlParse.
Now the new jsonParse is here too.
I didn't see any reason to create a new library ...